lineapy/api/apis.py

```diff
@@ -21,12 +21,12 @@
     get_slice_graph,
```

Comment:
I would rename this file `api_classes.py` or split the file into `artifact.py` and `artifact_store.py`. It's a little confusing to have both `api.py` and `apis.py`